torch/jit/frontend.py

```diff
@@ -544,6 +647,12 @@ def build_Continue(ctx, stmt):
     @staticmethod
     def build_With(ctx, stmt):
         r = ctx.make_range(stmt.lineno, stmt.col_offset, stmt.col_offset + len("with"))
+        # Handle ignore context manager
+        if is_torch_jit_ignore_context_manager(stmt):
+            if sys.version_info < (3, 9):
```

Comment:
Is there a work-around or a different library to use instead of `ast.unparse` so that we can support <3.9?

Reply:
Work around is to use astunparse library, but you will have to install it to use it. Not sure if we want that?

Reply:
Is `astunparse` completely equivalent to `ast.unparse`?

I am weighing two sides here:

Upon launch only some users who can afford to update to 3.9+ enjoys this feature

vs

Using `astunparse` to allow all users to enjoy this feature but we risk causing some functionality differences when migrating from `astunparse` to `ast.unparse` when our python support horizon moves past 3.9

Reply:
Yeah I think they are same. I was more worried about adding new dependency into the system. Do you think it is fine?